server/public/styles.css

```diff
@@ -0,0 +1,88 @@
+body {
+  background-color: #131425;
+}
+
+input[type="file"],
+input[type="submit"] {
+  display: none;
+}
+
+.flex-center {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.flex-center-column {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  flex-direction: column;
+}
+
+.fa-face-smile {
+  color: #34a604;
+  margin: 2rem 0;
+}
+
+.fa-image,
+.fa-check {
+  color: #808080;
+  cursor: pointer;
+}
+
+.check-holder {
+  margin: auto;
+  margin-top: 1.5rem;
+  max-width: 10rem;
+}
+
+.fa-image {
+  position: relative;
+  top: 40%;
+  margin: auto;
+}
+
+.fa-check {
+  color: #808080;
+  border: 2px solid #808080;
+  border-radius: 50%;
+  padding: 0.75rem;
+  transition: 0.25s;
+}
+
+.fa-check:hover {
+  color: #34a604;
+  border: 2px solid #34a604;
+}
```

Comment:
Instead of augmenting the font awesome classes, please define new ones to use in tandem with them.